schemas/browsers.schema.json

```diff
@@ -80,6 +80,10 @@
           "type": "boolean",
           "description": "Whether the browser supports extensions."
         },
+        "implements_webdriver_bidi": {
```

Comment:
Nit: can we use `"accepts_webdriver_bidi"` to match `"accepts_webextensions"`?

Reply:
How about a `has` array enum field that supports values `flags`, `webextension` and `webdriver-bidi`?

Reply:
See: https://github.com/mdn/browser-compat-data/pull/25825

Reply:
@queengooborg Renamed to `accepts_webdriver_bidi` in 5c5c1863d now.